source/agora/utils/PrettyPrinter.d

```diff
@@ -222,6 +222,9 @@ private struct TransactionFmt
         enum InputPerLine = 3;
         enum OutputPerLine = 3;
 
+        formattedWrite(sink, "Type : %s\n",
+            this.value.type);
+
         formattedWrite(sink, "Inputs (%d): %(%(%s, %),\n%)\n",
```

Comment:
```suggestion
        formattedWrite(sink, "Type: %s, Inputs (%d): %(%(%s, %),\n%)\n",
```

The goal is to have it be compact, as we might print a lot of data otherwise.

Reply:
Removed line Feed.